Formula/convco.rb

```diff
@@ -0,0 +1,39 @@
+class Convco < Formula
+  desc "Conventional commits, changelog, versioning, validation"
+  homepage "https://convco.github.io"
+  url "https://github.com/convco/convco/archive/refs/tags/v0.4.0.tar.gz"
+  sha256 "88c92b175163c8847da7dd201d32106c51ff85e2c992c2a3ff67e29ecbe57abb"
+  license "MIT"
+  head "https://github.com/convco/convco.git", branch: "master"
+
+  depends_on "rust" => :build
+  depends_on "libgit2@1.5"
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
```

Comment:
We don't need the `on_linux` block, since `rust` already depends on `pkg-config`